### DISCUSSION TITLE
Application Access - Openshift/OKD

### DISCUSSION THREAD

**Discussion by jasminstrkonjic** [gravitational/teleport#13180] [Q&A] [answered]
Hello Teleport Community!

Is it possible to set application access for Openshift or OKD platform?
We run Openshift and OKD on-prem and evaluating Teleport solution. SSH is working great but most of developers are accessing container platform through web browser for various reasons. 
Is it possible to add application access that way so we can monitor privilege accounts through web access on the platform? 

This would be very useful if someone causes an incident so we can address what is happening back to the source of the problem.
Since everything is contanerized I do not see a way of running an agent that will be able to access openshift console.

BR,
Jasmin

**Comment by jasminstrkonjic** [comment]
Nevermind I am able to setup application access, only have some issuess regarding ssl handshake. Will sort that out.

Does Teleport support application access recordings?
SSh sessions are recorded but applications are only logged not recorded.

Anyone knows is this supported?

BR,
Jasmin

**Reply by webvictim** [reply]
Teleport does record all the HTTP requests associated with an application access session, but they aren't currently exposed via the UI. There's an explanation of how to access the logs here: https://github.com/gravitational/teleport/discussions/11334#discussioncomment-2415029

**Reply by jasminstrkonjic** [reply]
Thank you! Hope it will be available in next releases.

**Comment by webvictim** [comment, marked as answer]
Teleport does record all the HTTP requests associated with an application access session, but they aren't currently exposed via the UI. There's an explanation of how to access the logs here: https://github.com/gravitational/teleport/discussions/11334#discussioncomment-2415029
